src/scale.rs

```diff
@@ -65,7 +65,7 @@ impl<T: Clone + One + Div<T, Output = T>, Src, Dst> Scale<T, Src, Dst> {
     /// The inverse Scale (1.0 / self).
     pub fn inv(&self) -> Scale<T, Dst, Src> {
         let one: T = One::one();
-        Scale::new(one / self.get())
+        Scale::new(one / self.0.clone())
```

Comment:
I find the previous version more readable.

Reply:
This is just preparation to change `&self` to `self`. `get` still require `Clone` bound (although at my sight, I would completely delete that method)